internal/platform/db/db.go

```diff
@@ -81,7 +81,7 @@ func (db *DB) Copy() *DB {
 
 // Execute is used to execute MongoDB commands.
 func (db *DB) Execute(ctx context.Context, collName string, f func(*mgo.Collection) error) error {
-	ctx, span := trace.StartSpan(ctx, "platform.DB.Execute")
+	_, span := trace.StartSpan(ctx, "platform.DB.Execute")
```

Comment:
Not sure I agree with this change

Reply:
Yes, you have a point here. I will rollback this change shortly.